<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Загрузка тарифов для Смоленской области</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>12</y>
     <width>458</width>
     <height>480</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>импортировать из</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtFileName">
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnSelectFile">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnView">
        <property name="text">
         <string>Просмотр</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>с  настройками из</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtCustomFileName">
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnCustomSelectFile">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCustomView">
        <property name="text">
         <string>Просмотр</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="chkOnlyForCurrentOrg">
      <property name="toolTip">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Импортируются только те тарифы, для которых код в поле &lt;span style=&quot; font-weight:600;&quot;&gt;MCOD &lt;/span&gt;источника совпадает с кодом &lt;span style=&quot; font-weight:600;&quot;&gt;ИНФИС &lt;/span&gt;текущей организации.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>только для текущего ЛПУ</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="gbDublicates">
          <property name="title">
           <string>Совпадающие записи</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QRadioButton" name="chkUpdate">
             <property name="text">
              <string>Обновить</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="chkAppend">
             <property name="toolTip">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Добавляется &lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;новый&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt; тариф, действующий с&lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt; текущей&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt; даты, а существующий тариф закрывается с днём ранее &lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;текущей&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt; даты.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Дополнить</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="chkSkip">
             <property name="text">
              <string>Пропустить</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="chkAskUser">
             <property name="text">
              <string>Спросить у пользователя</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lblUetValue">
        <property name="toolTip">
         <string>только для стоматологии</string>
        </property>
        <property name="text">
         <string>Стоимость одного УЕТ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="edtUetValue">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="maximum">
         <double>99999999.989999994635582</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="log"/>
    </item>
    <item>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnImport">
        <property name="text">
         <string>начать импортирование</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="labelNum">
        <property name="text">
         <string>всего записей в источнике:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnClose">
        <property name="text">
         <string>закрыть</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
